### PR TITLE
Fix : Make the tasks section of the workflows and best practices page more interactive

### PR DESCRIPTION
**Pull request by oyenuga17**
**This pull request makes the following changes:**
* Fixes issue #64 

**Checklist:**
- [x] File or folder now contains changes as specified in the issue i worked on
- [x] I have linked the issue I worked on to this pull request submitted by me
- [x] I certify that I ran my checklist.

Ping @Ifycode/git-github-training-resource

### CODE REVIEW

**Comment by oyenuga17** [comment]
I made changes please review @Ifycode

**Comment by Ifycode** [comment]
The issue you commented on, is not the issue you fixed @oyenuga17
This may just be the reason for your confusion too.

If someone else comments on this issue (you didn't comment on but) you submitted a PR for, I will pick their pull request over yours. That is the implication.

**Comment by Ifycode** [comment]
Don't close this pull request though, just leave it open. 

But first submit a pull request for the issue you commented on.

**Comment by oyenuga17** [comment]
oh I am sorry I forgot to show interest to this issue but it is the issue I fixed ..the first one I showed interest I could not understand it after I showed interest so I  just left it I did not create a pull request for it(i mean the first one) ..

so how do I go about it?

**Comment by Ifycode** [comment]
I left you 2 comments on that issue @oyenuga17. Check the comments: https://github.com/Ifycode/git-github-training-resource/issues/65#issuecomment-1290605779
Please do not write replies for that issue on this PR since they are unrelated. If you have any comments or questions for that issue, type them on that issue (since you have not submitted a PR for it yet). Not here.

File: doc/part-1-github-ui/workflows-and-best-practices.md [hunk 1]
```diff
@@ -26,6 +26,10 @@ Workflow common practices and best practices
 
 ## Tasks
 
+You will now learn to use everything you have learnt about in the learning videos. You should not have any issues while working on this task, if you follow the instructions to the latter. But incase you run into any issues, feel free to ask for help in the [Github discussion for part 1.](https://github.com/Ifycode/git-github-training/discussions/79) Or you can send the pull request anyway, and state the issue you have there in your pull request.
+
+**Task**: Follow the instructions in the issue created from the template in the tasks section of the [github contributions page.](https://obiagba-mary.gitbook.io/git-and-github-training/part-1-github-ui/github-contributions). This time, the you are to interact with the develop branch as shown in the learning videos.
+
```
Comment (Ifycode) [inline doc/part-1-github-ui/workflows-and-best-practices.md:32]: As for this PR, nice job adding the correct content. But you did not add the hint too. Add the hint also. You can read the related issue again to confirm.

**Review by Ifycode — CHANGES_REQUESTED** [review]

File: doc/part-1-github-ui/workflows-and-best-practices.md (hunk 1 above)
Comment (oyenuga17) [reply doc/part-1-github-ui/workflows-and-best-practices.md:32]: Made changes please review

File: doc/part-1-github-ui/workflows-and-best-practices.md [hunk 2]
```diff
@@ -26,6 +26,15 @@ Workflow common practices and best practices
...
+{% hint style="info" %}
+Note
+Interested contributors should read up How to submit your changes to this training resource for guidelines on [how to make contributions to this repository](https://obiagba-mary.gitbook.io/git-and-github-training/contributors-guide/making-and-submitting-changes)
+{% endhint %}
+
```
Comment (Ifycode) [inline doc/part-1-github-ui/workflows-and-best-practices.md:36]: Madam, what is contributors-related hint doing inside tasks section? 😂

Change the the hint to the correct hint. Again, see the related issue to know what hint to use.
You don't have to do it tonight. Rest if you need to. You will be able to think clearly and see your mistake 👍

**Review by Ifycode — CHANGES_REQUESTED** [review]

File: doc/part-1-github-ui/workflows-and-best-practices.md (hunk 1 above)
Comment (oyenuga17) [reply doc/part-1-github-ui/workflows-and-best-practices.md:32]: I am made another change please review

File: doc/part-1-github-ui/workflows-and-best-practices.md [hunk 3]
```diff
@@ -26,6 +26,14 @@ Workflow common practices and best practices
 
 ## Tasks
 
+You will now learn to use everything you have learnt about in the learning videos. You should not have any issues while working on this task, if you follow the instructions to the latter. But incase you run into any issues, feel free to ask for help in the [Github discussion for part 1.](https://github.com/Ifycode/git-github-training/discussions/79) Or you can send the pull request anyway, and state the issue you have there in your pull request.
+
+**Task**: Follow the instructions in the issue created from the template in the tasks section of the [github contributions page.](https://obiagba-mary.gitbook.io/git-and-github-training/part-1-github-ui/github-contributions). This time, the you are to interact with the develop branch as shown in the learning videos.
+
+{% hint style="success" %}
+The review you will get after you are done with the task will be based on the instructions in the issue and everything treated in the learning videos you have watched so far. Watch the video below to get better understanding on the tasks :thumbsup:
+{% endhint %}
```
Comment (Ifycode) [inline doc/part-1-github-ui/workflows-and-best-practices.md:35]: Looks good 👍

**Review by Ifycode — APPROVED** [review]

**Pull request merged**